api/lib/domain/usecases/create-pro-organizations-with-tags.js

```diff
@@ -36,16 +44,18 @@ module.exports = async function createProOrganizationsWithTags({
       organizationsToCreate,
       domainTransaction
     );
-    const organizationsTags = createdOrganizations.flatMap(({ id, externalId }) => {
+
+    const organizationsTags = createdOrganizations.flatMap(({ id, externalId, name }) => {
       return organizationsData.get(externalId).tags.map((tagName) => {
         const foundTag = allTags.find((tagInDB) => tagInDB.name === tagName.toUpperCase());
         if (foundTag) {
           return new OrganizationTag({ organizationId: id, tagId: foundTag.id });
         } else {
-          throw new OrganizationTagNotFound();
+          throw new OrganizationTagNotFound(`Le tag ${tagName} de l'organisation ${name} n'existe pas.`);
```

Comment:
C'était pas noté dans la description de la PR mais comme j'ai vu cette modification de code j'ai testé aussi => niquel ✅ 
<img width="776" alt="Capture d’écran 2022-05-03 à 15 30 59" src="https://user-images.githubusercontent.com/58915422/166462359-0a3efd24-bc87-4252-9713-e75a3da82769.png">

Reply:
Oui, j'en ai profité en passant pour rendre ce message plus explicite pour débugger quand on a le souci